.travis.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -e
+
+
+if [[ "${VIBED_SSL:-openssl}" == botan ]]; then
+    grep -qF "Have_botan" dub.sdl || echo 'versions "Have_botan"' >> dub.sdl
+    grep -qF 'dependency "botan"' dub.sdl || echo 'dependency "botan" version="~>1.12.0"' >> dub.sdl
```

Comment:
Is this still needed?

Reply:
Yep, this avoids consecutive updates when this script is rerun.